jqassistant.properties

```diff
@@ -1,15 +1,21 @@
 category=External Analyzers
 description=Analyze issues reported by jQAssistant.
 homepageUrl=https://github.com/jqassistant-contrib/sonar-jqassistant-plugin
-archivedVersions=1.7.3,1.8.1
-publicVersions=1.8.2
+archivedVersions=1.7.3,1.8.1,1.8.2
+publicVersions=1.9.0
  
 defaults.mavenGroupId=org.jqassistant.contrib.sonarqube
 defaults.mavenArtifactId=sonar-jqassistant-plugin
 
+1.9.0.description=Create ad-hoc rules and external issues instead of generic concept/constraint issues where possible and detect reports from parent modules
+1.9.0.date=2020-10-13
+1.9.0.sqVersions=[7.9,LATEST]
+1.9.0.changelogUrl=https://github.com/jqassistant-contrib/sonar-jqassistant-plugin/blob/master/release-notes.adoc#190
+1.9.0.downloadUrl=https://search.maven.org/remotecontent?filepath=org/jqassistant/contrib/sonarqube/sonar-jqassistant-plugin/1.9.0/sonar-jqassistant-plugin-1.9.0.jar
+
 1.8.2.description=Fixed JAXB related regression when running an analysis using Java version 11 or newer
 1.8.2.date=2020-02-24
-1.8.2.sqVersions=[7.9,LATEST]
+1.8.2.sqVersions=[7.9,8.1]
```

Comment:
This end range value needs to be the current version, 8.5. Otherwise your current users on 8.2+ won't be prompted by the Marketplace to upgrade.

Reply:
Thanks, fixed it!